jsonschema2pojo-gradle-plugin/example/android/app/build.gradle

```diff
@@ -29,11 +29,26 @@ android {
         }
     }
 
+    packagingOptions {
+        exclude 'META-INF/LICENSE.txt'
+    }
+
 }
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.android.support:appcompat-v7:21.0.3'
+    compile 'com.android.support:appcompat-v7:23.1.1'
+
+    // Required for @Generated annotation
+    compile 'org.glassfish:javax.annotation:10.0-b28'
```

Comment:
I don't understand why this is required. @Generated is part of Java's core: http://docs.oracle.com/javase/6/docs/api/javax/annotation/Generated.html